projects/ffmpeg/build.sh

```diff
@@ -0,0 +1,242 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+export OUT="/"
+export LINK_LIBS="-ldrm -lm -ldl -lXext -lz -lpthread -lrt -L/ffmpeg_deps/lib 
+                  -L/ffmpeg_deps/lib/alsa-lib/smixer/ -L/ffmpeg_deps/lib/vdpau/ \
+                  -I/ffmpeg_deps -I/ffmpeg \
+                  -lfdk-aac -lvorbisenc -lvorbisfile -l:smixer-hda.a -l:smixer-ac97.a \
+                  -l:smixer-sbase.a -lvdpau_trace -lavfilter -lavdevice -lpostproc \
+                  -lva -logg -lswscale -ltheoradec -ltheoraenc -lvpx -lasound -lbz2 \
+                  -lswresample -lvorbis -lvdpau -lavutil -lavcodec -lavformat \
+                  -ltheora -lva-drm -lxml2 -lopus -fuse-ld=/usr/bin/ld.lld"
+
+if [[ $CONFIG = "libfuzzer" ]]
+then
+      export CC="clang"
+      export CXX="clang++"
+      export CFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+      export CXXFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,float-divide-by-zero -fno-sanitize=vptr"
+      export LINK_FLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero $LINK_LIBS"
+      export TARGET_BSF="/ffmpeg/tools/target_bsf_fuzzer.c -o /target_bsf_fuzz"
+      export TARGET_DEC="-lxcb -lxcb-shm -lxcb -lxcb-xfixes -lxcb -lxcb-shape -lxcb -lX11 \
+                         -DFFMPEG_CODEC=AV_CODEC_ID_MPEG1VIDEO -DFUZZ_FFMPEG_VIDEO \
+                         /ffmpeg/tools/target_dec_fuzzer.c -o /target_dec_fuzz"
+      export TARGET_DEM="-DIO_FLAT=0 /ffmpeg/tools/target_dem_fuzzer.c -o /target_dem_fuzz"
+
+      mkdir -p $OUT/lib/
+      cp /usr/lib/x86_64-linux-gnu/libbz2.so.1.0 $OUT/lib/
+      cp /usr/lib/x86_64-linux-gnu/libz.so.1 $OUT/lib/
+      bzip2 -f -d alsa-lib-1.1.0.tar.bz2
+fi
+
+if [[ $CONFIG = "afl" ]]
+then
+      export CC="afl-clang-fast"
+      export CXX="afl-clang-fast++"
+      export CFLAGS="-g -fsanitize=address,integer,bounds,null,undefined,float-divide-by-zero"
+      export CXXFLAGS="-g -fsanitize=address,integer,bounds,null,float-divide-by-zero -fno-sanitize=vptr"
+      export LINK_FLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero $LINK_LIBS"
+      export TARGET_BSF="/ffmpeg/tools/target_bsf_fuzzer.c -o /target_bsf_afl"
+      export TARGET_DEC="-lxcb -lxcb-shm -lxcb -lxcb-xfixes -lxcb -lxcb-shape -lxcb -lX11 -lbz2 \
+                         -DFFMPEG_CODEC=AV_CODEC_ID_MPEG1VIDEO -DFUZZ_FFMPEG_VIDEO \
+                         /ffmpeg/tools/target_dec_fuzzer.c -o /target_dec_afl"
+      export TARGET_DEM="-DIO_FLAT=0 /ffmpeg/tools/target_dem_fuzzer.c -o /target_dem_afl"
+fi
+
+if [[ $CONFIG = "sydr" ]]
+then
+      export CC=clang
+      export CXX=clang++
+      export CFLAGS="-g"
+      export CXXFLAGS="-g -fno-sanitize=vptr"
```

Comment:
For Sydr and coverage targets you can remove `-fno-sanitize=vptr`, because there are no sanitizers in flags at all.

Reply:
done